modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala

```diff
@@ -264,4 +264,10 @@ object EpicsUtil {
     }).dropWhile(_.remaining.self.value === 0.0) // drop leading zeros
       .takeThrough(_.remaining.self.value > 0.0) // drop all tailing zeros but the first one
 
+  def removePartName(s: String): String = {
```

Comment:
Could you document this a bit? The code is clear but not why do you need to remove a part